tests/integ/modin/groupby/test_groupby_named_agg.py

```diff
@@ -55,3 +63,74 @@ def test_valid_func_with_named_agg_errors(basic_snowpark_pandas_df):
         assert_exception_equal=False,  # There is a difference in our errors.
         expect_exception_type=TypeError,
     )
+
+
+# Query count is 2 in below test due to line 65, basic_snowpark_pandas_df.to_pandas()
+# That produces the native pandas DataFrame to test against.
+@sql_count_checker(query_count=2)
+def test_named_agg_output_column_order(basic_snowpark_pandas_df):
+    eval_snowpark_pandas_result(
+        basic_snowpark_pandas_df,
+        basic_snowpark_pandas_df.to_pandas(),
+        lambda df: df.groupby("col1").agg(
+            new_col1=("col1", min), new_col2=("col2", min), new_col3=("col1", max)
+        ),
+    )
+
+
+# Query count is 2 in below test due to line 65, basic_snowpark_pandas_df.to_pandas()
+# That produces the native pandas DataFrame to test against.
+@sql_count_checker(query_count=2)
+def test_named_agg_output_column_order_with_dup_columns(basic_snowpark_pandas_df):
```

Comment:
Do you know what happens if you specify the same kwarg twice, like `df.agg(new_col1=("col1", min), new_col1=("col2", min))`? Is this worth adding a negative test for?

Reply:
Its a syntax error:
```python
In [5]: df.agg(new_col1=("col1", min), new_col1=("col2", min))
  Cell In[5], line 1
    df.agg(new_col1=("col1", min), new_col1=("col2", min))
                                   ^
SyntaxError: keyword argument repeated: new_col1


In [6]: pd.DataFrame(df).agg(new_col1=("col1", min), new_col1=("col2", min))
  Cell In[6], line 1
    pd.DataFrame(df).agg(new_col1=("col1", min), new_col1=("col2", min))
                                                 ^
SyntaxError: keyword argument repeated: new_col1
```
so I don't think it makes sense adding a test for it?

Reply:
Makes sense.